client/device/cloudftd/cloudftdonboarding/create.go

```diff
@@ -126,6 +126,7 @@ func Create(ctx context.Context, client http.Client, createInp CreateInput) (*Cr
 		ctx,
 		fmcconfig.UntilCreateDeviceRecordSuccess(ctx, client, createDeviceInp, &createOutp),
 		retry.NewOptionsBuilder().
+			Message("waiting for create FMC device record to be done").
```

Comment:
Waiting for FTD device record to be created on cdFMC...

Reply:
done 👍